hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/admin/om/GetServiceRolesSubcommand.java

```diff
@@ -70,10 +84,30 @@ private void getOmServerRoles(List<ServiceInfo> serviceList) {
       if (omRoleInfo != null &&
           serviceInfo.getNodeType() == HddsProtos.NodeType.OM) {
         System.out.println(
-            serviceInfo.getOmRoleInfo().getNodeId() + " : " +
-                serviceInfo.getOmRoleInfo().getServerRole() + " (" +
+            omRoleInfo.getNodeId() + " : " +
+                omRoleInfo.getServerRole() + " (" +
                 serviceInfo.getHostname() + ")");
       }
     }
   }
+
+  private void getOmServerRolesAsJson(List<ServiceInfo> serviceList)
```

Comment:
minor nit:  `printOmServerRolesAsJson` fits better. same for above method too

Reply:
Thank you for the suggestion, I have taken it up in the next commit